e2e/playwright/.auth/certified-user.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
this shouldn't be in this PR because it is not related to the issue.
please remove this. 
same goes for the changes made to the `index.html` file inside the `playwright-report`

Reply:
<img width="1040" alt="image" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/62249815/c17ab90c-1c8f-415a-a889-c0edbf7bb48b">

I tried to use the "discard changes" but it says "personal access token required". I do have SSH keys configured on GitHub. What should I do?